src/components/Home/Home.js

```diff
@@ -13,6 +16,27 @@ const Typography = withStyles({
 })(MuiTypography);
 
 export default function Home() {
+	const data = useStaticQuery(graphql`
+		{
+			catImages: allFile(filter: {relativePath: {glob:"cats/*"}}) {
```

Comment:
space after glob:

Reply:
You can't put string literal in constant here due to the way gatsby (and also tagged template) works. However, you _can_ consider using a page query – doesn't make a difference though.